RELEASES.md

```diff
@@ -1,3 +1,68 @@
+Version 1.22.0 (2017-11-23)
+==========================
+
+Language
+--------
+- [`non_snake_case` lint now allows extern no-mangle functions][44966]
+- [Now accepts underscores in unicode escapes][43716]
+- [`#![feature(const_fn)]` is now no longer required for using
+  const functions][43017] Still required for creating constant functions.
+
+Compiler
+--------
+- [rustc now defaults to having 32 codegen units at debug][44853]
```

Comment:
[It got reduced to 16](https://github.com/rust-lang/rust/pull/45064) in fact. On master, its also [disabled on some targets](https://github.com/rust-lang/rust/pull/45032), but the second PR didn't make it into beta.